CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 Changes by Version
 ==================
 
+1.37.0 (2022-08-11)
+-------------------
+
+* Upgrade operator-sdk to 1.22.2 ([#2021](https://github.com/jaegertracing/jaeger-operator/pull/2021), [@iblancasa](https://github.com/iblancasa))
+<<<<<<< HEAD
```

Comment:
It seems the merge was not done properly. Could you take a look? @rubenvp8510

Reply:
Oops, forgot to save the file. Fixed

Thanks!